packages/editor/src/components/Editor2Container.tsx

```diff
@@ -149,6 +149,13 @@ const EditorContainer = () => {
 
   return (
     <main className="pointer-events-auto">
+      <MetaTags>
+        <link
+          href="https://fonts.googleapis.com/css2?family=Figtree:ital,wght@0,300..900;1,300..900&display=swap"
+          rel="stylesheet"
+          type="text/css"
+        />
+      </MetaTags>
```

Comment:
we should add this in client folder
will merge put in this a later PR

Reply:
this doesnt cover the entire project, we need to put it in the head